pod2man.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+pod2man_helper() {
+    local podfile="${1?}"
+    local manfile="${2?}"
+    local outpath="${3?}"
+    local center="${4?}"
+    pod2man --section 8 --release --center "${center?}" --name "${manfile?}" "${podfile?}" "${outpath?}/share/man/${manfile?}.8"
+}
+
+main() {
+    local outpath="${1?}"
+    mkdir -p "${outpath?}/share/man" && \
+        type -p pod2man &> /dev/null && \
+        pod2man_helper "man/puppet-db.pod" "puppet-db" "${outpath?}" "manages PuppetDB administrative tasks" && \
+        pod2man_helper "man/puppet-query.pod" "puppet-query" "${outpath?}" "queries PuppetDB data" && \
+        pod2man_helper "man/puppetdb_conf.pod" "puppetdb_conf" "${outpath?}" "PuppetDB CLI configuration"
+}
+
+DIR="$(dirname "$0")"
+main "${1:-"${DIR?}"/target/debug}"
```

Comment:
we pass this in as an argument in vanagon https://github.com/puppetlabs/puppet-client-tools-vanagon/pull/12